tests/console/runc.pm

```diff
@@ -7,7 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Test runc installation and extended usage
+# Summary: Test docker-runc installation and extended usage
```

Comment:
The runc test doesn't test docker-runc ;)

Reply:
I've just dropped entirely the runc test and replaced it with docker-runc.